.solcover.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    skipFiles: ['GenericOracle.sol']
-};
\ No newline at end of file
+  skipFiles: ['GenericOracle.sol'],
```

Comment:
```suggestion
  istanbulReporter: ["html", "lcov"],
  skipFiles: ['GenericOracle.sol','test'],
```
why excluding GenericOracle ?

Reply:
this is not my changes, this PR only changes the code formatting (not the content)